systemtest/src/main/java/io/strimzi/systemtest/utils/specific/KeycloakUtils.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.utils.specific;
+
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.resources.ResourceManager;
+import io.strimzi.test.TestUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class KeycloakUtils {
+
+    private static final Logger LOGGER = LogManager.getLogger(KeycloakUtils.class);
+
+    private KeycloakUtils() {}
```

Comment:
Do we really need this?

Reply:
Yes, I am ensuring that object can not be created outside of class because it will be useless to just create object which has only static methods for my point of view.

Reply:
I agree it is useless to instantiate this class. But not sure we really need the constructor. I think that we have a lot of `*Utils` classes in the code with static methods which don't do this.